### PR TITLE
Add a basic ContentLocale method to controller

### PR DESCRIPTION
**Pull request by TheBnl**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10080

Add a basic ContentLocale method to controller that doesn't check a dataRecord.

### CODE REVIEW

File: src/Control/Controller.php [hunk 1]
```diff
@@ -655,6 +656,20 @@ public function redirectedTo()
     {
         return $this->getResponse() && $this->getResponse()->getHeader('Location');
     }
+    
+    /**
+     * Returns an RFC1766 compliant locale string, e.g. 'fr-CA'.
+     *
+     * Suitable for insertion into lang= and xml:lang=
+     * attributes in HTML or XHTML output.
```
Comment (michalkleiner) [inline src/Control/Controller.php:664]:
```suggestion
     * Note: The method is overloaded in {@link ContentController} where it takes into account
     * the current data record (@link SiteTree) and adds support for {@link Translatable}.
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]
I don't think Controller.php is really intended to serve content, that's what ContentController.php is for.


Reading through the issue, `login-forms` is a bit of a special case

I'm thinking a better approach would be to add the `ContentLocale()` method you've created and add it to https://github.com/silverstripe/silverstripe-login-forms/blob/4/src/EnablerExtension.php which is applied to Security.php

**Comment by michalkleiner** [comment]
@TheBnl I've moved the issue to the login-forms module and I'll close this issue here.
Can you please create a new PR there with the method added to the EnablerExtension as @emteknetnz suggests? Thanks!

**Comment by michalkleiner** [comment]
Moved issue — https://github.com/silverstripe/silverstripe-login-forms/issues/100

**Pull request closed without merging**